common/src/main/java/com/sequenceiq/cloudbreak/util/ConditionBasedEvaluatorUtil.java

```diff
@@ -41,4 +41,6 @@ private ConditionBasedEvaluatorUtil() {
         }
     }
 
+    public static void throwIfTrue(boolean empty) {
+    }
```

Comment:
Apart from the empty implementation (no `throw`), I wonder if this new overloaded variant is used anywhere.